changelog.md

```diff
@@ -13,7 +13,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fetch restaurants from sigarra and stored locally
 - Dropdown to choose the student's faculty in the login page
 - Calendar button to add an event to the calendar with the details of a exam
+- A Canteen Page in the personal area
 - Drag icon appears on widgets during editing mode
+- A Canteen Page in the personal area
+
```

Comment:
The changelog entry is repeated, can you fix it? You also need to update the pubspec version. Then ping me on slack and I'll merge this for you